launcher/service.go

```diff
@@ -72,13 +74,6 @@ func (launcher *Launcher) cacheCurrentServices(instances []InstanceInfo) {
 		}
 
 		launcher.currentServices[instance.ServiceID] = &service
-
-		if service.err == nil {
```

Comment:
why did you remove UseService, in this function the timestamp of the service is set, which affects the TTL of the removal of the service and the layer ?